extensions/2.0/Vendor/EXT_feature_metadata/examples/weather/weather.gltf

```diff
@@ -0,0 +1,219 @@
+              "valueType": "STRING"
+            },
+            "population": {
+              "name": "Population",
+              "valueType": "FLOAT32"
+            }
+          }
+        }
+      },
+      "featureCollections": {
+        "weatherCollection": {
+          "featureClass": "weather",
+          "featureCount": 1000,
+          "properties": {
+            "airTemperature": {
+              "elementCount": 1000,
+              "bufferView": 2
+            },
+            "airPressure": {
+              "elementCount": 1000,
+              "bufferView": 3
+            },
+            "windVelocity": {
+              "elementCount": 1000,
+              "bufferView": 4
+            }
+          }
+        },
+        "townCollection": {
+          "featureClass": "town",
+          "featureCount": 3,
+          "properties": {
+            "name": {
+              "elementCount": 3,
+              "bufferView": 5,
+              "offsetsBufferView": 6
+            },
+            "population": {
+              "elementCount": 3,
+              "bufferView": 7
+            }
+          }
+        }
+      }
+    }
+  },
+  "scene": 0,
+  "scenes": [
+    {
+      "name": "Scene",
+      "nodes": [
+        0
+      ]
+    }
+  ],
+  "nodes": [
+    {
+      "name": "Point Cloud",
+      "mesh": 0
+    }
+  ],
+  "meshes": [
+    {
+      "name": "Point Cloud Mesh",
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 0,
+            "_FEATURE_ID_0": 1
+          },
+          "mode": 0,
+          "extensions": {
+            "EXT_feature_metadata": {
+              "featureMappings": [
+                {
+                  "featureCollection": "weatherCollection",
+                  "featureIds": {
+                    "constant": 0,
+                    "vertexStride": 1
+                  }
+                },
+                {
+                  "featureCollection": "townCollection",
+                  "featureIds": {
+                    "attribute": "_FEATURE_ID_0"
+                  }
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "accessors": [
+    {
+      "name": "Positions",
+      "bufferView": 0,
+      "byteOffset": 0,
+      "componentType": 5126,
+      "count": 1000,
+      "max": [
+        0.9984954469408285,
+        0.9919821104589452,
+        0.9970798239119962
+      ],
+      "min": [
+        -0.9968233643517341,
+        -0.9973561515575708,
+        -0.9991830031102058
+      ],
+      "type": "VEC3"
+    },
+    {
+      "name": "Town Feature ID",
+      "bufferView": 1,
+      "byteOffset": 0,
+      "componentType": 5121,
+      "count": 1000,
+      "type": "SCALAR"
+    }
+  ],
+  "bufferViews": [
+    {
+      "name": "Positions",
+      "buffer": 0,
+      "byteLength": 12000,
+      "byteOffset": 0
+    },
+    {
+      "name": "Town Feature ID",
+      "buffer": 1,
+      "byteLength": 12000,
```

Comment:
`byteLength` isn't correct here

Reply:
something tells me I made a typo in my script... I'll double check.

Reply:
yup it was a typo, the property was declared as FLOAT32 when it should have been UINT16. I fixed the declaration.

Reply:
Er wait nevermind, I fixed the population one, not the feature ID.

Reply:
Oh I see... copy/paste error, I had one buffer that was a vec3 so the array was 3 times as long. In the case of town ID I had made it 3 times as long as I needed.